frontend/package.json

```diff
@@ -3,16 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
+    "@mui/material": "^6.1.3",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.3.3",
```

Comment:
do we need both material and bootstrap?

Reply:
I have removed the Material UI dependency and kept Bootstrap. The changes have been updated in commit [dec1eaa].